graphene_sqlalchemy/converter.py

```diff
@@ -159,11 +161,15 @@ def convert_enum_to_enum(type, column, registry=None):
     return lambda: enum_for_sa_enum(type, registry or get_global_registry())
 
 
-# TODO Make ChoiceType conversion consistent with other enums
```

Comment:
I think we should still plan to convert ChoiceType enums like other we do with other SQLAlchemy enums column types. That is:
- we should follow the same [consistent naming scheme](https://github.com/graphql-python/graphene-sqlalchemy/blob/9a0f7400957c5b5f5f8e952db2502ef1f79fac1f/graphene_sqlalchemy/enums.py#L9-L42)
- we should [register it in the registry](https://github.com/graphql-python/graphene-sqlalchemy/blob/9a0f7400957c5b5f5f8e952db2502ef1f79fac1f/graphene_sqlalchemy/enums.py#L45-L55) so we can later access it (see [example](https://github.com/graphql-python/graphene-sqlalchemy/blob/db3e9f4c3baad3e62c113d4a9ddd2e3983d324f2/graphene_sqlalchemy/tests/test_query_enums.py#L23-L24))

Would you mind leaving the TODO?